test_unstructured/metrics/test_text_extraction.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from unstructured.metrics.text_extraction import calculate_edit_distance
+from unstructured.partition.auto import partition
+
+
+@pytest.mark.parametrize(
```

Comment:
can we add more tests? a few im thinking of (can be existing files or new ones):
- test where unstructured output has spaced out words
- test where unstructured output has strung together words
- test where entire sentences are missing
- test with duplicated text

Reply:
We could, but what actually do we want to test against? Do we want to compare the score for those cases aka test the function scoring system? If file open and partitioning is not needed, I'd say we might not want to test against any files but just comparing mock string to mock string (which is partially captured in the first test).

Reply:
We want to test against a sample source cct string. This test does not need to test file opening and partitioning - yes, let's just compare mock strings (the CCT output)

Reply:
In your initial test there are a few arbitrary sample files, but the test cases I listed are specific occurrences in CCT strings we want to observe / test the scoring metric for. We can just make some sample strings to represent these common occurrences and the source string will be how we actually expect/want them to be parsed out of a document.

Reply:
Fyi for the above comment - I tested with `\n` and the score is actually worse because the original .txt file for fake-text is double spacing. 

I do believe that the score now is not as important as we don't have the actual test files just yet. From what I understand, we will want to use this metric especially when we make improvements on the partitioning function, therefore the process of partitioning should also be in the test case in the final version. But for now, we can test on mock strings.

From your previous comment, I believe you want to test the scoring function on different scenarios:
1. Same string but spaced out words
2. Same string but not spaces
3. Missing sentences
4. Duplicated text
Which the expected score could be manually calculated or using the very same Levenshtein distance calculation and put in.